pgjdbc/src/test/java/org/postgresql/test/jdbc2/TimezoneCachingTest.java

```diff
@@ -34,93 +34,94 @@ public void testPreparedStatementCachedTimezoneInstance() throws SQLException {
     Timestamp ts = new Timestamp(2016 - 1900, 1 - 1, 31, 0, 0, 0, 0);
     Date date = new Date(2016 - 1900, 1 - 1, 31);
     Time time = new Time(System.currentTimeMillis());
+    TimeZone tz = TimeZone.getDefault();
     PreparedStatement pstmt = null;
     try {
       pstmt = con.prepareStatement("INSERT INTO testtz VALUES (?,?)");
-      assertTrue(
+      assertEquals(
           "Cache never initialized: must be null",
-          getTimeZoneCache(pstmt) == null);
+          getTimeZoneCache(pstmt), null);
```

Comment:
Please flip arguments around.
The proper order is `assertEquals(String message, Object expected, Object actual)`